### PR TITLE
Update docs to mark compute_http(s)_health_check as legacy resources

### PR DESCRIPTION
**Pull request by danielcompton**
See https://cloud.google.com/compute/docs/load-balancing/health-checks#legacy_health_checks
for more info.

### CODE REVIEW

File: website/docs/r/compute_http_health_check.html.markdown [hunk 1]
```diff
@@ -10,11 +10,13 @@ description: |-
 
 Manages an HTTP health check within GCE. This is used to monitor instances
 behind load balancers. Timeouts or HTTP errors cause the instance to be
-removed from the pool. For more information, see [the official
-documentation](https://cloud.google.com/compute/docs/load-balancing/health-checks)
+removed from the pool. Creates an HttpHealthChecks resource. For more information, see [the official
```
Comment (paddycarver) [inline website/docs/r/compute_http_health_check.html.markdown]: This addition seems confusing to me, and doesn't really clarify anything. I think we could do without it.

File: website/docs/r/compute_https_health_check.html.markdown [hunk 2]
```diff
@@ -10,11 +10,13 @@ description: |-
 
 Manages an HTTPS health check within GCE. This is used to monitor instances
 behind load balancers. Timeouts or HTTPS errors cause the instance to be
-removed from the pool. For more information, see [the official
+removed from the pool. Creates an HttpHealthChecks resource. For more information, see [the official
```
Comment (paddycarver) [inline website/docs/r/compute_https_health_check.html.markdown]: Same here, this addition doesn't really clarify anything for me, and is more confusing than anything.

Comment (danielcompton) [reply website/docs/r/compute_https_health_check.html.markdown]: Fixed.

File: website/docs/r/compute_http_health_check.html.markdown (hunk 1 above)
Comment (danielcompton) [reply website/docs/r/compute_http_health_check.html.markdown]: Fixed.

Comment (paddycarver) [reply website/docs/r/compute_http_health_check.html.markdown]: Think you may have forgotten to push a commit?

Comment (danielcompton) [reply website/docs/r/compute_http_health_check.html.markdown]: Sorry, pushed now.

**Pull request merged**